doc/docportal/settings/cloud.rst

```diff
@@ -6,12 +6,12 @@ Use the Cloud tab to manage connected cloud storage services.
 
 For a comprehensive look at how to use ScummVM's built-in cloud functionality, check out our :doc:`../use_scummvm/connect_cloud` guide.
 
-From the Launcher, select **Options**, click the **>** scroll arrow until the Cloud tab is visible, and then select the **Cloud** tab.
+From the Launcher, select **Global Options**, if necessery click the **>** scroll arrow until the Cloud tab is visible, and then select the **Cloud** tab.
```

Comment:
Typo, should be `necessary`

Reply:
> If these are going to be the official docs for 2.6.0, they probably should say `2.6.0` instead (compare with `2.2.0` in many of the ones being replaced). I think it looks more professional that way as well since we're not using a daily build.

I wasn't really intending these to be the docs for 2.6.0 necessarily - if they're going to be then I'll need a hand to create a pull request against branch-2-6. Not sure how to do that without making a mess of things! 

With regards to the screenshots, up until now (disregarding the initial v2.2 docs), screenshots have not been updated with each release. In an ideal world, any PR making changes to the GUI would have accompanying new screenshots and documentation. This would result in docs and screenshots that are always up to date in terms of functionality, without the screenshot launcher versions necessarily matching - but as you say, this does look less professional. The alternative is that prior to each release I create an entirely new set of screenshots, which would probably look best, but is obviously the most labour intensive option!

Reply:
> I wasn't really intending these to be the docs for 2.6.0 necessarily - if they're going to be then I'll need a hand to create a pull request against branch-2-6. Not sure how to do that without making a mess of things!

We can just cherry-pick this to branch-2-6 after it's merged.

> With regards to the screenshots, up until now (disregarding the initial v2.2 docs), screenshots have not been updated with each release. In an ideal world, any PR making changes to the GUI would have accompanying new screenshots and documentation. This would result in docs and screenshots that are always up to date in terms of functionality, without the screenshot launcher versions necessarily matching - but as you say, this does look less professional. The alternative is that prior to each release I create an entirely new set of screenshots, which would probably look best, but is obviously the most labour intensive option!

Maybe we can just hide the version number from the screenshots?

Reply:
>Maybe we can just hide the version number from the screenshots?

Maybe just blurring the version number would work?
![launcher_list_group](https://user-images.githubusercontent.com/64062738/177108253-0e8508e5-71c3-4c17-bcf7-301d1743993d.png)